### PR TITLE
feat: release 2.0.0

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](/okta/okta-oidc-js/blob/master/CONTRIBUTING.md#commit)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Adding Tests
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: 277766

## What is the new behavior?
See CHANGELOG

## Does this PR introduce a breaking change?
- [x] Yes
- [ ] No

## Reviewers
@swiftone 
@robertjd

### CODE REVIEW

File: packages/okta-angular/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+
+### Bug Fixes
+
+- Saved URI is now stored in `sessionStorage` instead of `localStorage`. This fixes an issue which can occur when multiple instances of the app are loading at the same time.
+- `OktaCallbackComponent` uses `window.location.replace()` to complete the login flow after `handleAuthentication` completes. This fixes an issue where the user could navigate back to the callback hander.
+
+### Breaking Changes
+
+- Signature for `onAuthRequired` callback functions has changed. Callbacks will receive the `OktaAuthService` as the first argument, and the Angular `Injector` as the second argument.
+- Static initializer `OktaAuthModule.initAuth()` has been removed. `OKTA_CONFIG` should be provided directly by your module.
```
Comment (swiftone) [inline packages/okta-angular/CHANGELOG.md]: I'm not up on my angular usages, but does this impact any patterns where instantiation and initialization are different steps?  I'd expect not, but Angular loves to do somethings in ways I don't expect.

File: packages/okta-angular/package.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@okta/configuration-validation": "^0.4.1",
-        "@okta/okta-auth-js": "^2.11.2",
+        "@okta/okta-auth-js": "3.0.0",
```
Comment (swiftone) [inline packages/okta-angular/package.json]: Why exact?

File: packages/okta-angular/yarn.lock [hunk 3]
```diff
@@ -412,16 +412,15 @@
...
+  version "3.0.0"
+  resolved "https://registry.yarnpkg.com/@okta/okta-auth-js/-/okta-auth-js-3.0.0.tgz#8fc9d07e3e2906f6f2506fa92a483789892110fb"
+  integrity sha512-RD9R2JdNYKeg4OD6weRk/tHURVbC88L40ve2qZdbi1UrfdLyh3wlmrz5H/9H80my1t5JACZvUIR9MWnQaEapGA==
   dependencies:
     Base64 "0.3.0"
     cross-fetch "^3.0.0"
     js-cookie "2.2.0"
     node-cache "^4.2.0"
-    q "1.4.1"
-    reqwest "2.0.5"
```
Comment (swiftone) [inline packages/okta-angular/yarn.lock]: This makes me happy

File: packages/okta-angular/CHANGELOG.md (hunk 1 above)
Comment (aarongranick-okta) [reply packages/okta-angular/CHANGELOG.md]: We have had this deprecated for awhile, and removed it from the documentation some time ago. The static initializer was incompatible with AOT compilation.

File: packages/okta-angular/package.json (hunk 2 above)
Comment (aarongranick-okta) [reply packages/okta-angular/package.json]: Because I typed it in manually. I'll update to a flexible semver

**Comment by swiftone** [comment]
Quick request - when you merge, can you make the commit message mention _okta-angular_ v 2.0?

**Pull request merged**